src/Symfony/Component/Translation/Dumper/JsonFileDumper.php

```diff
@@ -23,9 +23,23 @@ class JsonFileDumper extends FileDumper
     /**
      * {@inheritdoc}
```

Comment:
@ogizanagi 
this code inherit my previous, accepted PR:
https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Translation/Dumper/FileDumper.php